assets/js/modules/analytics/components/common/PropertySelectIncludingGA4.js

```diff
@@ -91,7 +93,20 @@ export default function PropertySelectIncludingGA4() {
 				return;
 			}
 
-			trackEvent( 'analytics_setup', 'property_change', newPropertyID );
+			if ( newPropertyID === PROPERTY_CREATE ) {
+				trackEvent(
+					`${ viewContext }_analytics`,
+					'change_property_new',
+					'ua'
+				);
+			} else {
+				const label = !! internalID ? 'ga4' : 'ua';
```

Comment:
The internal property ID is available for UA properties. So it should be other way around:
```suggestion
				const label = !! internalID ? 'ua' : 'ga4';
```